lib/consts/ChainType.ts

```diff
@@ -0,0 +1,35 @@
+export enum ChainType {
+  LTC = 'LTC',
+  BTC = 'BTC',
+}
+
+export type Info = {
```

Comment:
I would put just the `enum` in this file the the `type`s back into the `ChainClient.ts` file

Reply:
It creates a Travis error: `loop import`.